src/main/java/io/github/syst3ms/skriptparser/util/SkriptDate.java

```diff
@@ -7,43 +7,32 @@
 import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
-import java.time.ZoneId;
 import java.util.Locale;
 import java.util.TimeZone;
 
-/**
- * Represents a date.
- * @author Njol
- */
 public class SkriptDate implements Comparable<SkriptDate> {
     // TODO make a config for this
     public final static String DATE_FORMAT = "EEEE dd MMMM yyyy HH:mm:ss.SSS zzzXXX";
     public final static Locale DATE_LOCALE = Locale.US;
-    @SuppressWarnings("FieldMayBeFinal")
-    private static ZoneId ZONE_ID = ZoneId.systemDefault();
+    private static TimeZone TIME_ZONE = TimeZone.getDefault();
```

Comment:
why not setting visibility to public and then dont use reflexion in test (bellow on the PR) ?

Reply:
Because then people would be able to change it without any good explanation. There is a public method that returns this field already. Reflection being the requirement means that it normally shouldn't be changed.

Reply:
Mea culpa i based this on the fact that  millis in hours field is public but isn't a big problem if user change it